user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1427,6 +1429,11 @@ define([
       }, 100);
     },
 
+    /*
```

Comment:
Please, use the special comment start used for licenses: `/*!`. It is recognized and preserved by minification tools.